src/components/Applications/Applications.tsx

```diff
@@ -26,15 +36,39 @@ interface Props {
 interface State {
   currentApplicationId: string | undefined,
   currentApplicationFilename: string | undefined,
-  documents: DocumentInformation[]
+  documents: DocumentInformation[],
+  openModal: Boolean | false,
 }
 
 class Applications extends Component<Props, State, {}> {
   ButtonFormatter = (cell, row) => (
     <div>
       <Link to="/applications/send">
-        <button type="button" className="btn btn-primary w-75 btn-sm p-2 m-1" onClick={(event) => this.handleViewDocument(event, row)}>View Application</button>
+        <button type="button" className="btn btn-primary w-75 btn-sm p-2 m-1" onClick={(event) => this.placeholderFunc}> <b>Apply Now</b></button>
+      </Link>
+    </div>
+  )
+
+  DoubleButtonFormatter = (cell, row) => (
+    <div className="double-button-jawn">
+      <Link to="/applications/send">
+        <button type="button" className="btn btn-primary w-110 btn-sm p-2 m-1" onClick={(event) => this.placeholderFunc}> <b>Download</b></button>
```

Comment:
can we add the fetch call to download the specific application? we already have a download api route